### DISCUSSION TITLE
Userset rewrites plans

### DISCUSSION THREAD

**Discussion by costap** [ory/keto#603] [Ideas]
Hi folks

I know of https://github.com/ory/keto/issues/263 to implement this feature, I was wondering if you know more or less when would be able to implement it and if you have any thoughts of what will it look like?

Thanks
Pedro

**Comment by vinckr** [comment]
I cant comment on the actual plans, but there already was a bit of discussion in this issue: https://github.com/ory/keto/issues/466 . 
But feel free to continue here, just posting this in case someone might have missed it.
